docs/denoising.rst

```diff
@@ -0,0 +1,250 @@
+This is colloquially known as "non-aggressive denoising".
+
+However, users may wish to apply a different type of denoising,
+or to incorporate other regressors into their denoising step,
+and we will discuss these alternatives here.
+
+``tedana`` uses independent component analysis (ICA) to decompose the data into components
+which each have a spatial map of weights and time series that are assumed to reflect meaningful underlying signals.
+It then classifies each component as "accepted" (BOLD-like) or "rejected" (non-BOLD-like).
+The data are denoised by taking the time series of the rejected components and regressing them from the data.
+
+``tedana`` uses a spatial ICA, which means the components are `statistically independent` across space, not time.
+That means the time series from accepted and rejected components can share variance.
+If a rejected component shares meaningful variance with an accepted component,
+then regressing the rejected components' time series from your data may also remove meaningful signal
+associated with the accepted component as well.
+Depending on the application,
+people may take different approaches on how to handle variance that is shared between accepted and rejected components.
+These different approaches towards decomposition-based denoising methods are described here.
+
+This page has three purposes:
+
+1.  Describe different approaches to denoising using ICA components.
+2.  Provide sample code using common code to perform each type of denoising.
+3.  Describe how to incorporate external regressors (e.g., motion parameters) into the denoising step.
+
+.. admonition:: Which should you use?
+
+  The decision about which denoising approach you should use depends on a number of factors.
+
+  The first thing to know is that aggressive denoising, while appropriate for orthogonal time series,
+  may remove signal associated with "good" time series that are independent to, but not orthogonal with,
+  the "bad" time series being regressed out.
+
+  As an alternative, users may want to try non-aggressive denoising or aggressive denoising combined with component orthogonalization.
+
+  The main difference between the orthogonalization+aggressive and non-aggressive approaches is that,
+  with orthogonalization,
+  all the variance common across the accepted and rejected components is put in the accepted basket,
+  and therefore it is not removed in the nuisance regression.
+  In contrast, with the non-aggressive denoising,
+  the shared variance is smartly split during the estimation process and therefore some amount is removed.
+
+Now we can get started!
+This walkthrough will show you how to perform different kinds of denoising on example data.
+
+For this walkthrough, we will use Python and the Nilearn package to denoised data.
+However, you can definitely accomplish all of these steps in other languages, like MATLAB,
+or using other neuroimaging toolboxes, like AFNI.
+
+.. important::
```

Comment:
I think you can remove this "Important" section. These are file names, for the same of the demo, there's no reason to say why these specific file names were used. The note on using different regressors is also indirectly mentioned elsewhere.

Reply:
Good catch. I'll remove it.